src/Program.cs

```diff
@@ -26,9 +28,22 @@ internal class Program
         internal const int UnableToLocateDotNetCliExitCode = 4;
 
         private static async Task<int> Main(string[] args)
+        {
+            var rootCommand = CreateCommandLineOptions();
+
+            // Parse the incoming args and invoke the handler
+            return await rootCommand.InvokeAsync(args);
+        }
+
+        public static RootCommand CreateCommandLineOptions()
         {
             var rootCommand = new RootCommand
             {
+                new Argument<string>("project")
```

Comment:
The argument needs to be named differently from "workspace". Otherwise `System.CommandLine` can not call `Program.Run`. *project* may not be the best name. Alternatives:
* work-space
* solution-or-project

Reply:
project will be fine for the time being. Once we flip `--folder` to being a bool flag we will probably want to revisit the name since it could be a project, solution, or folder path.